src/views/ContactPage/ContactPage.jsx

```diff
@@ -4,6 +4,7 @@ import ContactSection from "../../components/contacComponents/contactSection/Con
 import MapContainer from "../../components/contacComponents/googleMap/GoogleMap";
```

Comment:
import `GoogleMap` as `GoogleMap` (not as `MapContainer`) - it helps with readability of the code, cause automatically we are looking for a component with this name